app/controllers/application_controller.rb

```diff
@@ -1,2 +1,40 @@
 class ApplicationController < ActionController::Base
+  before_action :set_default_meta_tags
+
+  private
+
+  def set_default_meta_tags
+    title = 'Hotwire.io'
+    description = "Community-driven documentation and resource Hub for Hotwire."
+    image = "#{root_url}og-image.png"
+
+    tags = {
+      site: title,
+      reverse: true,
+      separator: '·',
+      description: description,
+      keywords: ['hotwire', 'stimulus.js', 'turbo drive', 'turbo streams', 'turbo frames', 'documentation', 'community'],
+      twitter: {
+        title: title,
+        description: description,
+        card: 'summary_large_image',
+        image: image,
+      },
+      og: {
+        title: title,
+        description: description,
+        type: 'website',
+        url: root_url,
```

Comment:
I think here we want something like:
```suggestion
        url: request.original_url,
```

Or why would we want to point to the `root_url` here?